src/safeposix/syscalls/net_calls.rs

```diff
@@ -100,66 +118,66 @@ impl Cage {
     pub fn bind_syscall(&self, fd: i32, localaddr: &interface::GenSockaddr) -> i32 {
         self.bind_inner(fd, localaddr, false)
     }
+    fn bind_inner_socket(&self, sockfdobj: &mut SocketDesc, localaddr: &interface::GenSockaddr, prereserved: bool) -> i32 {
```

Comment:
same?

Reply:
For this one given that this function is a factored out version of the previous they're grouped together. If you'd rather it be otherwise then I'd change it.

Reply:
I think the fact that they're consecutive is fine enough to know they're grouped together. It's just a bit hard to read without spacing, though it is pretty pedantic.